workflows/subwf-atac.wdl

```diff
@@ -164,70 +117,63 @@ workflow wf_atac {
             quality_filter = quality_filter,
             bc_error_threshold = bc_error_threshold,
             bc_probability_threshold = bc_probability_threshold,
-            read_format = read_format
+            read_format = select_first([get_read_format.read_format, read_format])
```

Comment:
This might create problems in the future because we will never be able to use our own  read_format for share.
If we change something to the files in the future we will have to change the task to be able to test. Also, do we need still need linkers since we removed the shifting?

Reply:
What do you mean by "do we still need linkers"? You mean like to remove the linker sequences when writing the barcodes into the fastq files during demux?